resources/packaging/archlinux/PKGBUILD.in

```diff
@@ -30,11 +30,10 @@ source=(https://commondatastorage.googleapis.com/chromium-browser-official/chrom
         chromium-launcher-$_launcher_ver.tar.gz::https://github.com/foutrelis/chromium-launcher/archive/v$_launcher_ver.tar.gz
         chromium-$pkgver.txt::https://chromium.googlesource.com/chromium/src.git/+/$pkgver?format=TEXT
         #https://raw.githubusercontent.com/gcarq/inox-patchset/$pkgver/product_logo_{16,22,24,32,48,64,128,256}.png
```

Comment:
When I was creating this PKGBUILD from Inox's, I wasn't sure what to do with the product_logo images. If they're not needed, I think it's best to remove it and related code elsewhere to reduce clutter.

Does ungoogled-chromium still have an icon if these files are removed? If not, I believe there is an original Chromium logo in the build output that we can use.

Reply:
Checked and this is not needed. I'll delete the lines and add the icons and logos the same way official chromium PKGBUILD does (simpliy taking them from the source tree).